Components/TripPreview/TripPreview.js

```diff
@@ -1,49 +1,97 @@
 import React from 'react';
-import { Text, View, StyleSheet } from 'react-native';
+import { Text, View, StyleSheet, TouchableOpacity } from 'react-native';
 
```

Comment:
Regarding the destructuring of destinations from route.params, the Back End property is 'destination' (singular).
I'm not sure how everything comes together. Destinations makes more sense than destination. Should I ask them to amend it, or should we adhere to what's already built?


Here is a copy from the BE team's repo regarding their models (for reference)
```
class Destination(models.Model):
    location = models.CharField(max_length=100)
    abbrev = models.CharField(max_length=3)
    lat = models.CharField(max_length=100)
    long = models.CharField(max_length=100)
```

Reply:
I spoke with the BE team and they will be amending the property name from 'destination' to 'destinations'

I've included the updated model of the trip object below, though some aspects may be subject to change as they work through the details on their side.

Seems that each destination object within the destinations property will only have an abbrev property (we need to let them know if we want more information here)
```
    allTrips {
        id
        name
        origin
        originAbbrev
        originLat
        originLong
        startDate
        endDate
        user {
            id
            name
            email
        }
        **destinations**: [
            {
                abbrev
            }
       ]
        tripDestinationDates [{
            destination {
                name
                abbrev
            }
            date
        }]
    }
}